nixos/modules/services/misc/ethminer.nix

```diff
@@ -102,7 +104,7 @@ in
       script = ''
         ${pkgs.ethminer}/bin/.ethminer-wrapped \
           --farm-recheck ${toString cfg.recheckInterval} \
-          --report-hashrate \
+          --report-hr \
```

Comment:
Same here?

Reply:
looks like they added support for both versions of this flag: `--report-hashrate` and `--report-hr`. 
 this flag reports back to the mining pool what it thinks the current mining rate "should" be. https://github.com/ethereum-mining/ethminer/blob/master/ethminer/main.cpp#L263 i do think the original version is more legible and will revert this change.

Reply:
change reverted